react-app/src/styles/loginForms.css

```diff
@@ -0,0 +1,59 @@
+html {
+    font-size: 62.5%;
+}
+
+body {
+    background-color: #DEE2E6;
```

Comment:
This is color-light-3, right?

Reply:
Yeah, should be probably be replaced with the correct variable